Plots/XY/NCL_xy_35.py

```diff
@@ -0,0 +1,249 @@
+"""
+NCL_xy_35.py
+============
+This script illustrates the following concepts:
+   - Use of opacity resources
+     to control opacity of curves and markers in
+     XYPlots.
+   - How to set line opacity and marker opacity to 
+     different values using RGBA tuples
+   - Opacity is still functional in colorless plots
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/xy_35.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/xy_35_lg.png
+"""
+
+###############################################################################
+# Import packages:
+
+import numpy as np
+from matplotlib import pyplot as plt
+from matplotlib import colors
+
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Create data:
+
+# Make array of x-values, 64 evenly spaced values between 0 and 1
+f = np.linspace(0., 1., 64)
+
+twopi = 2 * 3.141592654
```

Comment:
This is probably sufficient for pi, but if you want to you could use the math package instead and use `math.pi`

Reply:
Or to avoid importing another package, I would use np.pi. Both have the same value, but like Rose said up to you!